kernel/packages/decentraland-ecs/types/@decentraland/PortableExperiences/index.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module '@decentraland/PortableExperiences' {
+  enum ExecutorType {
+    SCENE = 'SCENE',
+    WEARABLE = 'WEARABLE',
+    QUEST_UI = 'QUEST_UI'
+  }
```

Comment:
Enums are values, which cannot be used in .d.ts files, because enums are also values in typescript. Please use a union type 
```ts
type ExecutorType = 'SCENE' | 'WEARABLE' | 'QUEST_UI'
```